core/frontend/src/render/webgl/Target.ts

```diff
@@ -1284,6 +1301,16 @@ export class OnScreenTarget extends Target {
   public readImageToCanvas(): HTMLCanvasElement {
     return this._usingWebGLCanvas ? this.copyImageToCanvas() : this._2dCanvas.canvas;
   }
+
+  public get screenSpaceEffects(): Iterable<string> {
+    return this._screenSpaceEffects;
+  }
+
+  public set screenSpaceEffects(effects: Iterable<string>) {
+    this._screenSpaceEffects.length = 0;
+    for (const effect of effects)

```

Comment:
very minor, could replace this loop with:
```ts
this._screenSpaceEffects.push(...effects);
```
although I'm not sure what babel or tsc's down-transpilation will turn it into. Considering the input type is any iterable, it might be similar

Reply:
Replaced loop with `this._screenSpaceEffects = [...effects];`.